wails-frontend/src/pages/importFile.tsx

```diff
@@ -48,19 +50,37 @@ const ImportFile = () => {
   };
 
   return (
-    <section>
-      <div>{req.Msg}</div>
-      <div className="baseline">{baselineStr()}</div>
-      <div className="flex">
-        <button className="btn" onClick={handleCancel}>
-          Cancel
-        </button>
-        <button className="btn" onClick={handleApply}>
-          {applyStr()}
-        </button>
+    <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
+      <div className="w-1/4  max-w-sm  min-w-fit">
+        <div>
+          <p className="mas-title text-neutral pb-4">{req.Msg}</p>
+        </div>
+        <div>
+          <p
+            className={
+              account
+                ? ' mas-body text-s-success  pb-4'
+                : ' mas-body text-neutral pb-4'
+            }
```

Comment:
Can we have that extracted from the code to a const!? 🤔

Reply:
probably !

Reply:
Please, do it... closing here then!